src/Beman/Optional26/tests/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# cmake-format: off
+# src/Beman/Optional26/tests/CMakeLists.txt -*-makefile-*-
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# cmake-format: on
+
+include(GoogleTest)
+
+# Tests
+add_executable(optional_test
```

Comment:
The target should probably be called beman_optional26_test or something like that to avoid collisions in case this project is embedded in a larger project. Target names are global.

Reply:
We may want to use PROJECT_IS_TOP_LEVEL to gate that in order to be friendlier with Fetch. For packaging tests ought to be run, but for vendoring that needs to be more controllable. 

https://cmake.org/cmake/help/latest/variable/PROJECT_IS_TOP_LEVEL.html

Reply:
> The target should probably be called beman_optional26_test or something like that to avoid collisions in case this project is embedded in a larger project. Target names are global.

Done.

> We may want to use PROJECT_IS_TOP_LEVEL to gate that in order to be friendlier with Fetch. For packaging tests ought to be run, but for vendoring that needs to be more controllable.

I'm not sure what would be the improvement here. We already have conditional in parent CmakeLists.txt:
```shell
# in Beman/Beman.Optional26/src/Beman/Optional26/CMakeLists.txt

# Tests
if(BUILD_TESTING)
  add_subdirectory(tests)
endif()
```
@steve-downey , am I missing something?